app/src/main/res/values/arrays.xml

```diff
@@ -684,8 +684,8 @@
         <item>Blue Boost</item>
         <item>Colour Enhance</item>
         <item>Cyanisation</item>
-
-  </string-array>
+        <item>fade</item>
```

Comment:
`Fade`

Reply:
thank you @mayank8318

Reply:
a little help needed @mayank8318 ,from where should i made these changes.Do i need to close these PR and make new one again